src/entities/schedulerstate.ts

```diff
@@ -0,0 +1,23 @@
+/** @hidden */
+export class SchedulerState {
+    constructor(
+    /** Whether this entity exists or not */
+    public exists?: boolean,    // EntityExists
+
+    /**
+     * The serialized entity state. This can be stale while
+     * CurrentStateView != null.
+     */
+    public state?: string,      // EntityState
+
+    /** The queue of waiting operations, or null if none. */
+    public queue?: unknown,     // Queue
+    // TODO: what type should this be/is it ever used on this side?
```

Comment:
This class technically isn't used yet. I'm not sure if it will need to be used at all? It looks like the extension side of things does a lot of work to translate the `SchedulerState` into the relevant state for out-of-proc languages, which the `DurableEntityBindingInfo` class serves as in this repo. Any additional relevant state we need to pass (like locked entitites maybe?) we can just add to that class. 

I will get rid of this for now.